public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/controller.php

```diff
@@ -0,0 +1,323 @@
+<?php
+
+namespace WordCamp\Blocks\Schedule;
+
+use WP_Post, WP_REST_Request;
+use const WordCamp\Blocks\{ PLUGIN_DIR, PLUGIN_URL };
+
+defined( 'WPINC' ) || die();
+
+/**
+ * Register block types and enqueue scripts.
+ *
+ * @return void
+ */
+function init() {
+	$front_end_assets = require PLUGIN_DIR . 'build/schedule-front-end.min.asset.php';
+
+	wp_register_script(
+		'wordcamp-schedule-front-end',
+		PLUGIN_URL . 'build/schedule-front-end.min.js',
+		$front_end_assets['dependencies'],
+		$front_end_assets['version'],
+		true
+	);
+
+	wp_register_style(
+		'wordcamp-schedule-front-end',
+		PLUGIN_URL . 'build/schedule-front-end.min.css',
+		array(),
+		filemtime( PLUGIN_DIR . 'build/schedule-front-end.min.css' )
+	);
+
+	wp_set_script_translations( 'wordcamp-schedule-front-end', 'wordcamporg' );
+
+	$block_arguments = array(
+		'attributes'      => get_attributes_schema(),
+		'render_callback' => __NAMESPACE__ . '\render',
+
+		'editor_script' => 'wordcamp-blocks',
+		'editor_style'  => 'wordcamp-blocks',
+
+		/*
+		 * @todo This gets loaded on all front-end pages unnecessarily. See:
+		 *
+		 * https://github.com/WordPress/gutenberg/issues/15256
+		 * https://github.com/WordPress/gutenberg/issues/21838
+		 *
+		 * There isn't a simple workaround, though, since the block needs to be registered before the global $post
+		 * is available. We can probably just wait until that bug is fixed upstream. An alternative option might be:
```

Comment:
Does the render_callback method (the second link above) not work in this case? Register the scripts, then enqueue them in `render` (I haven't tried this, but it seems to work for other folks)

Reply:
I didn't look at that new one closely since I've been focused on the beta, but I just tested it and it looks like it works 👍 

0a0ba2da